CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 ## New features
 
+### CMake
+
+* `find_package(sEqAn3)` is now case-insensitive and always populates `SEQAN3_*` variables in all upper-case.
```

Comment:
```suggestion
* `find_package(SeqAn3)` is now case-insensitive and always populates `SEQAN3_*` variables in all upper-case.
```

Reply:
I thought it would be funny to show that we now accept any case.

Reply:
Reminds me of
<img src="https://user-images.githubusercontent.com/12967715/71084252-2a323a80-2195-11ea-8e23-daca6b77e590.jpg" width="300">
😆 

But it's kinda weird to read in the changelog and looks like a typo.